frontend/app/src/components/history/trades/ClosedTrades.vue

```diff
@@ -440,7 +319,7 @@ watch(loading, async (isLoading, wasLoading) => {
                     :disabled="selected.length === 0"
                     @click="massDelete"
                   >
-                    <v-icon> mdi-delete-outline </v-icon>
+                    <v-icon> mdi-delete-outline</v-icon>
```

Comment:
This doesn't seem like an intentional change

Reply:
reverted.